example/index.html

```diff
@@ -15,6 +15,14 @@
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.0/styles/dark.min.css"
     />
+    <script type="importmap">
+      {
+        "imports": {
+          "hello-csv": "https://cdn.jsdelivr.net/npm/hello-csv@0.1.1/dist/bundled/index.es.js"
```

Comment:
@emily-fields perhaps we could add to our deployment.mdx a step to update the version here?